README.md

```diff
@@ -113,6 +113,14 @@ The following Patterns would match it:
   }
 }
 ```
+```json
+{
+  "Image": {
+    "Thumbnail": {
+      "Url": [ "a", { "prefix": "https:" } ] }
```

Comment:
nit: 
```suggestion
      "Url": [ "a", { "prefix": "http:" } ] }
```

I guess you did that one on purpose to see if I'm reviewing :)

Reply:
Good catch!